config/src/lib.rs

```diff
@@ -298,6 +298,7 @@ impl Parameters {
 
 #[derive(Clone, Serialize, Deserialize, Eq, Hash, PartialEq, Debug)]
 pub struct WorkerInfo {
+    // pub public_key: PublicKey,
```

Comment:
Add a TODO to explain this maybe? Or was it meant to be deleted?

Reply:
ah thanks this should be removed